packages/core/core/src/Strapi.ts

```diff
@@ -30,14 +30,14 @@ import type {
   Schema,
 } from '@strapi/types';
 
-import loadConfiguration from './core/app-configuration';
+import loadConfiguration from './configuration';
```

Comment:
Should we take the opportunity to remove the default export here and make it a named export? That way it won't matter that the filename is different from what we're calling the import.

Reply:
I'll enable the no-default rule later and do a oneshot pass most likely